add node.js websockets reference docs (#446)
nitrictech/docs: src/pages/reference/nodejs/v0/websocket/websocket-close.mdx

export const description =
  "Reference for Nitric's Node.js library - Close a connection to a websocket"

# Node.js - websocket.close()

Closes a connection to a websocket

```javascript
import { websocket } from '@nitric/sdk'

const socket = websocket('socket')

/**
 * Close a connection to a client.
 *
 * @param connectionId the client received when connecting to the websocket
 */
const closeConnection = async (connectionId) => {
  await socket.close(connectionId)
}
```

## Parameters

<Properties>
  <Property name="connectionId" required type="string">

>     The ID of the connection which should be closed.

Reviewer: ```suggestion
    The ID of the connection to be closed.
```
